lib/base_helper.rb

```diff
@@ -385,12 +386,17 @@ def find_follow(*paths)
         end
       end
 
+      def compare_time(t1, t2)
```

Comment:
compare usually returns `0`, `1`, or `-1`. Also looking at the logic, a better name would be `compare_str_time`.
Instead of `1` or `-1` also just integer value should also be acceptable. You can `return tp2 - tp1` at the end.

Reply:
Thanks, will update.